tests/sentry/api/endpoints/test_group_external_issues.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import, print_function
+
+import six
+
+from sentry.models import PlatformExternalIssue
+from sentry.testutils import APITestCase
+from sentry.testutils.helpers import with_feature
+
+
+class GroupExternalIssuesEndpointTest(APITestCase):
+    @with_feature('organizations:sentry-apps')
+    def test_simple(self):
```

Comment:
How about naming this something more descriptive like `test_exposes_accessible_external_issues`?

I think we should move away from these generic test names that don't describe anything. Tests are a good place to learn about what an object is supposed to encapsulate and the interface they expose. Tests are basically the best docs on how an object interacts with the world and being descriptive helps, imo.

Reply:
Im gonna follow up on the other pr